docs/get-started/getting-started-aws-emr.md

```diff
@@ -0,0 +1,132 @@
+
+####  Step 4: Security
+
+Select an existing "EC2 key pair" that will be used to authenticate SSH access to the cluster's nodes. If you do not have access to an EC2 key pair, follow these instructions to [create an EC2 key pair](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html#having-ec2-create-your-key-pair).
+
+*Optionally* set custom security groups in the "EC2 security groups" tab.
+
+In the "EC2 security groups" tab, confirm that the security group chosen for the "Master" node allows for SSH access. Follow these instructions to [allow inbound SSH traffic](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/authorizing-access-to-an-instance.html) if the security group does not allow it yet.
+
+![Step 4: Security](pics/Rapids_EMR_GUI_4.PNG)
+
+#### Finish Cluster Configuration
+
+The EMR cluster management page displays the status of multiple clusters or detailed information about a chosen cluster. In the detailed cluster view, the "Summary" and "Hardware" tabs can be used to monitor the status of master and core nodes as they provision and initialize.
+
+When the cluster is ready, a green-dot will appear next to the cluster name and the "Status" column will display **Waiting, cluster ready**.
+
+In the cluster's "Summary" tab, find the "Master public DNS" field and click the `SSH` button. Follow the instructions to SSH to the new cluster's master node.
+
+![Finish Cluster Configuration](pics/Rapids_EMR_GUI_5.PNG)
+
+
+### Running an example joint operation using Spark Shell
+
+SSH to the EMR cluster's master node, get into sparks shell and run the sql join example to verify GPU operation.
+
+```
+spark-shell
+```
+
+Running following Scala code in Spark Shell
+
+```
+val data = 1 to 10000
+val df1 = sc.parallelize(data).toDF()
+val df2 = sc.parallelize(data).toDF()
+val out = df1.as("df1").join(df2.as("df2"), $"df1.value" === $"df2.value")
+out.count()
+```
+
+### Submit Spark jobs to a EMR Cluster Accelerated by GPUs
+
+Similar to spark-submit for on-prem clusters, AWS EMR supports a Spark application job to be submitted. The mortgage examples we use are also available as a spark application.  You can also use **spark shell** to run the scala code or **pyspark** to run the python code on master node through CLI.
+ 
+
+
+### Running GPU Accelerated Mortgage ETL and XGBoost Example using EMR Notebook
+
+An EMR Notebook is a "serverless" Jupyter notebook. Unlike a traditional notebook, the contents of an EMR Notebook itself—the equations, visualizations, queries, models, code, and narrative text—are saved in Amazon S3 separately from the cluster that runs the code. This provides an EMR Notebook with durable storage, efficient access, and flexibility.
+
+You can use the following step-by-step guide to run the example mortgage dataset using Rapids on Amazon EMR GPU clusters. For more examples, please refer to [NVIDIA/spark-rapids](https://github.com/NVIDIA/spark-rapids/)

```

Comment:
This link brings the user to the main page for the RAPIDS Accelerator source code, not to more examples.  @sameerz @krajendrannv do you have a more appropriate link you'd like to see here?

Reply:
yes @jlowe agreed. We need to point to the right ETL notebook and XGBoost Notebook